src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -9,9 +25,17 @@ namespace CommunityToolkit.Maui.Core.Views;
 /// </summary>
 public class MauiMediaElement : Grid, IDisposable
 {
+	bool navbarExist;
+	bool isDisposed;
+
+	static readonly AppWindow? appWindow = GetAppWindowForCurrentWindow();
 	readonly MediaPlayerElement mediaPlayerElement;
 
-	bool isDisposed;
+	ImageSource source { get; set; } = new BitmapImage(new Uri("ms-appx:///whitefs.png"));
```

Comment:
I think this one can be `static` and used across all MediaElement instances

```suggestion
	static readonly ImageSource source = new BitmapImage(new Uri("ms-appx:///whitefs.png"));
```

Reply:
The recommendation has been applied @pictos . Thanks for the suggestion. Marked as resolved.